packages/builtin/string.pony

```diff
@@ -101,14 +100,36 @@ actor Main
       _set(0, 0)
     else
       _size = len
+      _alloc = alloc.max(_size + 1)
```

Comment:
At first glance I think this does the wrong thing in the overflow case (`_size == USize.max_value()`).